lib/util/support.js

```diff
@@ -24,12 +24,24 @@ function isGetUserMediaSupported() {
     || !!(navigator.mozGetUserMedia);
 }
 
+/**
+ * @returns {boolean} - true if the browser is explicitely unsupported.
+ *  Note: this function returning false does not mean that browser is supported.
+ */
+function isExplicitelyUnsupportedBrowser() {
+  if (typeof navigator !== 'undefined' && typeof navigator.userAgent === 'string') {
+    return /Edge/.test(navigator.userAgent);
+  }
+  return false;
+}
+
 /**
  * Check if the current environment is supported by the SDK.
  * @returns {boolean}
  */
 function isSupported() {
-  return !!guessBrowser()
+  return !isExplicitelyUnsupportedBrowser()
+    && !!guessBrowser()
```

Comment:
@makarandp0 Wouldn't it be simpler to modify `guessBrowser()` to detect Edge? Then the above line can be refactored like so:
```suggestion
    && guessBrowser() && guessBrowser() !== 'edge'
```

Reply:
@manjeshbhargav  I thought about it initially, but decided against it because 
1) modifying `guessBrowser` requires changes to @twilio/webrtc - and  
2) `guessbrowser` is widely used, I didn't want it to affect any other callers. if it starts returning `edge` where it returned `chrome`, other callers would need to handle this case.